packages/ui-components/src/components/tag-alarm/tag-alarm.tsx

```diff
@@ -0,0 +1,51 @@
+import { isEmpty } from 'lodash-es';
+
+import { Component, Host, h, Prop } from '@stencil/core';
+
+import { EAlarmSeverity, ETagAlarmSize } from './tag-alarm.types';
+import { ALARM_CONFIG } from './tag-alarm.config';
+
+@Component({
+	tag: 'kv-tag-alarm',
+	styleUrl: 'tag-alarm.scss',
+	shadow: true
+})
+export class KvTagAlarm {
+	/** (required) Alarm Severity */
+	@Prop({ reflect: true }) severity!: EAlarmSeverity;
+	/** (optional) Tag Alarm Size */
+	@Prop({ reflect: true }) size?: ETagAlarmSize = ETagAlarmSize.Normal;
+	/** (optional) Hide alarm text */
+	@Prop({ reflect: true }) hideText?: boolean = false;
+	/** (optional) Alarm custom text */
+	@Prop({ reflect: true }) text?: string = '';
```

Comment:
Does this work?

```tsx
@Prop({ reflect: true }) text?: string = ALARMS_CONFIG[this.severity].name;
```

I just find ternary operators too confusing.

Reply:
No, the idea of the text prop is to allow other inputs like the alarm count.

Reply:
@djcouto this would work on first render and I believe it would cover all use cases. However, if you try this change on storybook, changing the severity will only update the colors and not the text. To fix that I believe it would be needed to add a watch on the severity variable to then update the text in case of changes.

Reply:
`label` is a better naming here I think. And same for `hideText` -> `hideLabel`